src/app/user/profile/profile.page.html

```diff
@@ -1,14 +1,68 @@
-<ion-header [translucent]="true">
-  <ion-toolbar>
-    <ion-title>Profile</ion-title>
-  </ion-toolbar>
-</ion-header>
-
 <ion-content [fullscreen]="true">
-  <ion-header collapse="condense">
-    <ion-toolbar>
-      <ion-title size="large">Profile</ion-title>
-    </ion-toolbar>
-  </ion-header>
-  <app-explore-container name="Profile"></app-explore-container>
+  <app-header />
+  <div collapse="condense">
+    <div class="heading-title">
+      <h1>Профил</h1>
+      <p>
+        {{user.role === "regular" ? 'на потребител' : "на организатор"}}
+      </p>
+    </div>
+  </div>
+  <div class="container">
+    <form #userEditForm="ngForm" (submit)="onUserEdit(userEditForm)">
+      <div class="auth-wrapper">
+        <div class="auth-input">
+          <ion-input type="email" label="Имейл*" labelPlacement="floating" name="email" ngModel [ngModel]="user.email"
+            #email="ngModel" clearInput required
+            pattern="^[a-zA-Z0-9\._+\-]+@[a-zA-Z0-9\.]+\.[a-zA-Z]{2,}$"></ion-input>
+        </div>
+
+        <div class="auth-input">
+          <ion-input type="password" label="Парола*" labelPlacement="floating" name="password" ngModel
```

Comment:
Add TODO for the password change overlay

Reply:
done